packages/form-control/README.md

```diff
@@ -0,0 +1,177 @@
+# @open-wc/form-control
+A standardized mixin for creating form-associated custom elements using a standardized validation function pattern.
+
+## Install
+
+```sh
+# npm
+npm install @open-wc/form-control
+
+# yarn
+yarn add @open-wc/form-control
+```
+
+## Usage
+
+After importing, create a web component class that extends the mixin, and provide your desired base class as the input to `FormControlMixin`.
+
+> The `FormControlMixin` has been tested with both [LitElement](https://lit.dev/) and `HTMLElement`, so `LitElement` is not required, but all examples in this documentation will show `LitElement` web component syntax and decorators.
+
+
+```typescript
+// custom web component class that extends FormControlMixin
+
+import { LitElement, html } from 'lit';
+import { customElement, query, property } from 'lit/decorators.js'
+import { live } from 'lit/directives/live.js';
+
+import { FormControlMixin } from '@open-wc/form-participation';
```

Comment:
This will be from `@open-wc/form-control` the way the project is currently structured.